package.json

```diff
@@ -4,28 +4,28 @@
   "version": "5.0.0",
   "description": "Use Remix React Router in your stories",
   "type": "module",
-  "packageManager": "pnpm@9.15.0",
+  "packageManager": "pnpm@10.15.1",
   "scripts": {
     "preinstall": "npx only-allow pnpm",
     "test": "vitest --run"
   },
   "devDependencies": {
     "@auto-it/conventional-commits": "^11.3.0",
-    "@typescript-eslint/eslint-plugin": "^5.61.0",
-    "@typescript-eslint/parser": "^5.61.0",
-    "@vitest/ui": "^3.1.3",
+    "@typescript-eslint/eslint-plugin": "^8.46.2",
+    "@typescript-eslint/parser": "^8.46.2",
+    "@vitest/ui": "^4.0.6",
     "auto": "^11.3.0",
-    "boxen": "^5.0.1",
-    "dedent": "^0.7.0",
-    "eslint": "^8.44.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-react-refresh": "^0.4.1",
-    "lint-staged": "^13.2.3",
-    "prettier": "2.8.8",
-    "typescript": "5.3.2",
-    "vite": "^6.0.3",
-    "vitest": "^3.1.3"
+    "boxen": "^8.0.1",
+    "dedent": "^1.7.0",
+    "eslint": "^9.39.0",
+    "eslint-config-prettier": "^10.1.8",
+    "eslint-plugin-prettier": "^5.5.4",
+    "eslint-plugin-react-hooks": "^7.0.1",
+    "eslint-plugin-react-refresh": "^0.4.24",
+    "lint-staged": "^16.2.6",
+    "prettier": "3.6.2",
+    "typescript": "5.9.3",
```

Comment:
I prefer keep the TS version a bit old, so we only create types that work with projects using older version of TS. That's why the version is pinned ;)